CODEOWNERS

```
@@ -18,111 +22,126 @@
 # @cilium/policy        Policy behaviour
 # @cilium/proxy         L7 proxy, Envoy
 # @cilium/vendor        Vendoring, dependency management
-#
-# Specific code owners:
-# @eloycoto             Debian packaging
-# @aanm @ianvernon      Docker packaging
-# @scanf @borkmann      BPF documentation
 
 # The following filepaths should be sorted so that more specific paths occur
 # after the less specific paths, otherwise the ownership for the specific paths
 # is not properly picked up in Github.
 * @cilium/janitors
-api/ @cilium/api
-bpf/ @cilium/bpf
-bugtool/cmd/ @cilium/cli
-cilium/ @cilium/cli
-cilium/cmd/preflight_k8s_valid_cnp.go @cilium/kubernetes
-cilium-health/cmd/ @cilium/cli
-cilium-health/launch/ @cilium/health
-contrib/packaging/deb/ @eloycoto
-contrib/packaging/docker/ @aanm @ianvernon
-contrib/vagrant @cilium/ci
-daemon/ @cilium/agent
-daemon/cmd/datapath.* @cilium/bpf
-daemon/cmd/endpoint.* @cilium/endpoint
-daemon/cmd/health.* @cilium/health
-daemon/cmd/hubble.go @cilium/hubble
-daemon/cmd/ipcache.* @cilium/ipcache
-daemon/cmd/loadbalancer.* @cilium/loadbalancer
-daemon/cmd/metrics.* @cilium/metrics
-daemon/cmd/policy.* @cilium/policy
-daemon/cmd/prefilter.go @cilium/bpf
-daemon/cmd/proxy.go @cilium/proxy
-daemon/cmd/state.go @cilium/endpoint
-daemon/cmd/sysctl_linux.go @cilium/bpf
-Documentation/ @cilium/docs
-Documentation/bpf.rst @scanf @borkmann
-Documentation/contributing/ @cilium/contributing
-envoy/ @cilium/proxy
-examples/ @cilium/docs
-examples/kubernetes/ @cilium/kubernetes
-examples/minikube/ @cilium/kubernetes
+/api/ @cilium/api
+/api/v1/flow/ @cilium/api @cilium/hubble
+/api/v1/observer/ @cilium/api @cilium/hubble
+/api/v1/peer/ @cilium/api @cilium/hubble
+/bpf/ @cilium/bpf
+Makefile* @cilium/build
+/bpf/Makefile* @cilium/build @cilium/bpf
+/bpf/sockops/Makefile* @cilium/build @cilium/bpf
+Dockerfile* @cilium/build
+/bugtool/cmd/ @cilium/cli
+/cilium/ @cilium/cli
+/cilium/cmd/preflight_k8s_valid_cnp.go @cilium/kubernetes
+/cilium-health/ @cilium/health
+/cilium-health/cmd/ @cilium/health @cilium/cli
+/contrib/packaging/ @cilium/build
+/contrib/vagrant/ @cilium/contributing
+/contrib/coccinelle/ @cilium/bpf
+/daemon/ @cilium/agent
+/daemon/cmd/datapath.* @cilium/bpf
+/daemon/cmd/endpoint.* @cilium/endpoint
+/daemon/cmd/health.* @cilium/health
+/daemon/cmd/hubble.go @cilium/hubble
+/daemon/cmd/ipcache.* @cilium/ipcache
+/daemon/cmd/loadbalancer.* @cilium/loadbalancer
+/daemon/cmd/metrics.* @cilium/metrics
+/daemon/cmd/policy.* @cilium/policy
+/daemon/cmd/prefilter.go @cilium/bpf
+/daemon/cmd/proxy.go @cilium/proxy
+/daemon/cmd/state.go @cilium/endpoint
+/daemon/cmd/sysctl_linux.go @cilium/bpf
+/Documentation/ @cilium/docs
+/Documentation/bpf.rst @cilium/bpf
+/Documentation/contributing/ @cilium/contributing
+/Documentation/envoy/ @cilium/envoy
```

Nit, there's no envoy team:

```suggestion
/Documentation/envoy/ @cilium/proxy
```